app/policies/kondo_policy.rb

```diff
@@ -0,0 +1,11 @@
+class KondoPolicy < ApplicationPolicy
+  class Scope < Scope
+    def resolve
+      scope.all
+    end
+
+    def index?
+      true
+    end
```

Comment:
We don't need these lines [as per the video at 57:00](https://kitt.lewagon.com/camps/718/lectures/05-Rails%2F07-Airbnb-Facebook-connect#video).

Reply:
Thanks for tip! I've updated the changes.